README.md

```diff
@@ -72,43 +86,53 @@ If the repositories of the subproject are under the WG's GitHub organization, th
 
 ### Meetings
 
-* Regular WG Meeting: Montly meetings, last wednesday of the Month at 2 pm UTC
+* Regular WG Meeting: Biweekly meetings, at 3 pm UTC
   * The meetings will be announced on discourse at least a week in advance
   * Recordings and meeting notes will be made available after each recording
+  * These announcements will be done on [the official ROS discourse with the wg-aerial-robotics tag](https://discourse.ros.org/tag/wg-aerial-robotics)
 
 ### Communication Channels
 
-For now, Discourse only, however we will discuss a chat based communication channel soon.
+Any news to be shared, meeting announcements or discussion points should be on the [Aerial Vehicles category on ROS discourse](https://discourse.ros.org/c/aerial-vehicles/).
+
+Quick chats, meeting reminders, subcommittee communication or just for fun sharing is on the [#cwg-aerial channel in the ROS discord Server](https://discord.com/channels/1077825543698927656/1141902822254850128)
+
 
 ### Backlog Management
 
-Github Projects
+[Github Projects](https://github.com/ROS-Aerial/community/projects?query=is%3Aopen)
 
 ### Membership, Roles and Organization Management
 
-Working Group members may act in one or more of the following roles:
-
-* **Member**
-  * Prerequisite: Attend at least one out of the last three Working Group meetings
-  * Responsible for triaging issues
-* **Reviewer**
-  * All reviewers are members
-  * Prerequisite: Proven track record of high-quality reviews to WG Subprojects
-  * Responsible for reviewing pull requests
-* **Approver**
-  * All approvers are reviewers
-  * Prerequisite: Proven track record of high-quality contributions and reviews to WG Subprojects
-  * Responsible for approving and merging pull requests
-  * Responsible for vetting and accepting new projects into the Working Group
-* **Lead**
-  * TSC member or their delegate
-  * Responsible for organizing and moderating working group meetings
-  * Responsible for posting meeting materials (minutes, recordings, etc.)
-  * Responsible for breaking ties
-
-To become a member or change role, create an issue in this repository using the appropriate issue template.
+Since the aerial ros working group is community driven, it will not follow the official guidelines. Currently of now we just have the following roles:
```

Comment:
```suggestion
Since the ROS Aerial working group is community-driven, it will not follow the official guidelines. Currently of now we just have the following roles:
```